frontend/app/src/components/home/RepliesTo.svelte

```diff
@@ -34,13 +34,17 @@
     $: displayName = me
         ? client.toTitleCase($_("you"))
         : client.getDisplayNameById(repliesTo.senderId, $communityMembers);
+    $: messageContext = {
```

Comment:
This worries me a little bit. Have you thought about the scenarios where the reply context is different from the current context. e.g. replying privately etc.